packages/interfaces/src/json-value.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 // source https://git.io/fAtwX
-export type JSONValue = string | number | boolean | JSONObject | JSONArray;
+export type JSONValue = string | number | boolean | null | JSONObject | JSONArray;
```

Comment:
Could you please add changes to [changelog](../changelog.md)?

Reply:
Done